### PR TITLE
Feature/install linkerd

### PR DESCRIPTION
**Pull request by girishranganathan**

### CODE REVIEW

File: linkerd/config_templates/emojivoto.yml [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+      creationTimestamp: null
+      labels:
+        app: emoji-svc
+    spec:
+      serviceAccountName: emoji
+      containers:
+      - env:
+        - name: GRPC_PORT
+          value: "8080"
+        image: buoyantio/emojivoto-emoji-svc:v7
```
Comment (leecalcote) [inline linkerd/config_templates/emojivoto.yml]: I wonder if we will want to unpin the tagged version at some future point. ":v7"

Comment (wmorgan) [reply linkerd/config_templates/emojivoto.yml]: Literally just released v8 yesterday :)

Comment (girishranganathan) [reply linkerd/config_templates/emojivoto.yml]:
```
Using default tag: latest
Error response from daemon: manifest for buoyantio/emojivoto-web:latest not found
```
I would have loved to just use it from the URL directly, which is what I tried first. The issue was the hardcoded namespace.
Please let me know if you guys know of an alternative url where the namespace is not hardcoded.

Comment (girishranganathan) [reply linkerd/config_templates/emojivoto.yml]: actually, never mind, I was able to force the namespace from the code. . . this is now removed

Comment (leecalcote) [reply linkerd/config_templates/emojivoto.yml]: That's too funny. :) Thanks for pointing this out, @wmorgan.

File: linkerd/install.go [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+	Assets     []*Asset `json:"assets,omitempty"`
+}
+
+type Asset struct {
+	Name        string `json:"name,omitempty"`
+	State       string `json:"state,omitempty"`
+	DownloadURL string `json:"browser_download_url,omitempty"`
+}
+
+func (iClient *LinkerdClient) getLatestReleaseURL() error {
```
Comment (grampelberg) [inline linkerd/install.go]: You might want to use https://versioncheck.linkerd.io/version.json instead. That'll help differentiate between stable and edge releases as well.

**Pull request merged**